src/App.js

```diff
@@ -1,28 +1,12 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
```

Comment:
- [ ] Kindly the App.js component should be class-based. As asked by the requirements.